src/OrchardCore.Modules/OrchardCore.Users/Views/UserFields.cshtml

```diff
@@ -1,6 +1,16 @@
+@using Microsoft.AspNetCore.Identity
+
 @model SummaryAdminUserViewModel
 
+@inject UserManager<IUser> UserManager
+
 <h5 class="float-start">@Model.User.UserName <span class="hint dashed">@Model.User.Email</span></h5>
+@if (!await UserManager.IsEmailConfirmedAsync(Model.User))
```

Comment:
I would do a null or empty string check before checking if the email is confirmed.

Reply:
This is doing the same as the action menu. No need for another check.

Reply:
Perhaps we should fix it in both places. Email is a string and is nullable. Creating a bad user will break the UI

Reply:
It's safe in both places. `IsEmailConfirmedAsync` is actually only checking the `EmailConfirmed` flag.